packages/cosmo-eccodes-definitions/package.py

```diff
@@ -39,6 +39,15 @@ class CosmoEccodesDefinitions(Package):
 
     depends_on('eccodes@2.14.1 ~aec', when='~aec')
 
+    def setup_run_environment(self, env):
+        eccodes_definition_path = self.spec['cosmo-eccodes-definitions'].prefix + '/cosmoDefinitions/definitions/:' + self.spec['eccodes'].prefix + '/share/eccodes/definitions/'
+        env.prepend_path('GRIB_DEFINITION_PATH', eccodes_definition_path)
+        eccodes_samples_path = self.spec['cosmo-eccodes-definitions'].prefix + '/cosmoDefinitions/samples/'
+        env.prepend_path('GRIB_SAMPLES_PATH', eccodes_samples_path)
```

Comment:
Can eccodes be run on its own? If not, I think it should only be in `setup_dependent_build_environment` or possibly `setup_dependent_run_environment` (if that function exists).

Reply:
setup_run_environment is per default > setup_dependent_run_environment since the run_env of the dependencies is always loaded whereas the build_env is not. Otherwise I don't understand your comment, do you mean that this should be in the eccodes package? I guess you are right, the problem is then I don't get the cosmo-eccodes-definitions.prefix. I guess I should just put a prepend_path('PATH',   self.spec['eccodes'].prefix) as well.

Reply:
No need of adding the prepend_path('PATH', self.spec['eccodes'].prefix) since it is actually automatically done.